### DISCUSSION TITLE
Impact after adding a GitHub organization to an Enterprise

### DISCUSSION THREAD

**Discussion by sputheti** [community/community#50346] [Enterprise]
### Select Topic Area

Question

### Body

Hello Team - We are considering upgrading our GitHub organization from the Team Plan to an Enterprise, and then adding Organization to an Enterprise; and we're curious about the potential impact on DevOps and Developers. Specifically, we'd like to know if there will be any changes to Repo URLs, Access tokens, Credentials, or other factors that could impact our CI/CD pipelines. Your insights would be greatly appreciated. Thank you

**Comment by Mohamed-SayedAlAhl** [comment]
When adding an organization to an Enterprise on GitHub, there could be some impact on DevOps and Developers. Here are a few potential changes that you might encounter:

1. Repo URLs: Adding an organization to an Enterprise might lead to changes in your repository URLs. If you are using absolute URLs in your CI/CD pipelines, you might need to update them to reflect the new URLs.

2. Access tokens: You might need to generate new access tokens for your CI/CD pipelines if the existing ones are tied to the previous organization or team plan.

3. Credentials: If you are using credentials that are specific to your previous organization or team plan, you might need to update them to work with the new organization and Enterprise.

4. Permissions: Adding an organization to an Enterprise might also lead to changes in permissions for your team members. You might need to adjust the permissions for repositories and other resources to ensure that your team members have the appropriate access.

It is important to carefully plan and test any changes to your CI/CD pipelines after adding an organization to an Enterprise. This can help you identify and address any potential issues before they affect your development and deployment processes.
